packages/client/components/PokerEstimateHeaderCardParabol.tsx

```diff
@@ -87,31 +99,65 @@ const Content = styled('div')({
   paddingRight: 4
 })
 
+const CardTitle = styled('h1')({
+  fontSize: 16,
+  lineHeight: '24px',
+  margin: '0 0 8px'
+})
+
+const CardTitleWrapper = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'flex-start',
+  width: '100%'
+})
+
+const CardDescription = styled('div')({
+  color: PALETTE.SLATE_700,
+  fontWeight: 'normal',
+  lineHeight: '20px',
+  fontSize: 14,
+  margin: 0,
+  height: '100%'
+})
+
 interface Props {
   stage: PokerEstimateHeaderCardParabol_stage
 }
 
-type Story = Required<NonNullable<NonNullable<PokerEstimateHeaderCardParabol_stage>['story']>>
-
 const PokerEstimateHeaderCardParabol = (props: Props) => {
   const {stage} = props
   const {story} = stage
-  const {content, id: taskId, teamId} = story as Story
-  const integration = story!.integration
   const atmosphere = useAtmosphere()
   const [isExpanded, setIsExpanded] = useState(false)
   const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
-  const [editorState, setEditorState] = useEditorState(content)
+  const [editorState, setEditorState] = useEditorState(story?.content)
   const editorRef = useRef<HTMLTextAreaElement>(null)
   const descriptionRef = useRef<HTMLDivElement>(null)
   const maxHeight = Math.min(descriptionRef.current?.scrollHeight ?? 300, 300)
   useEffect(
     () => () => {
       setIsExpanded(false)
     },
-    [taskId]
+    [story?.id]
   )
-  const {useTaskChild} = useTaskChildFocus(taskId)
+  const {useTaskChild} = useTaskChildFocus(story?.id || '')
```

Comment:
This feels ugly but if we check whether `story` exists beforehand and return the error card before `useEffect` or `useTaskChildFocus` are called, we won't be following the rules of hooks

Reply:
i _really_ wish the rule of hooks didn't apply to returning `null`.
the least ugly pattern i've found is `const storyId = story?.id ?? ''` at the top of the component

Reply:
Makes sense! I've moved it to the top which feels slightly less ugly